third_party/move/move-compiler-v2/src/lib.rs

```diff
@@ -570,7 +570,10 @@ fn report_bytecode_verification_error(
         let debug_info = if command_line::get_move_compiler_backtrace_from_env() {
             format!("\n{:#?}", e)
         } else {
-            "".to_string()
+            format!(
+                "\nError message: {}",
```

Comment:
Was this just for debugging?  I notice that the Error message is always none in the actual test cases.

Reply:
As said above,  this is intended to surface, and not just for debugging. Where there is 'none'  we have some more work to do.

That we see error messages at all is because `signature_checker_v2` was a rewrite by @vgao1996 a while ago with good error messages. This is the standard we should apply to the other verifier parts as well...